src/components/form/radio/radio.tsx

```diff
@@ -37,6 +38,10 @@ export interface RadioProps {
   checked?: boolean;
   disabled?: boolean;
   onChange: ChangeEventHandler<HTMLInputElement>;
+  /**
+   * Object of props passed to the <input/>
+   */
+  labelProps?: CommonProps & InputHTMLAttributes<HTMLInputElement>;
```

Comment:
And this should extend the <label> element not an Input element.
```suggestion
  /**
   * Object of props passed to the `<label />`
   */
  labelProps?: CommonProps & LabelHTMLAttributes<HTMLLabelElement>;
```